content/faq/41_what-if-i-could-not-find-anyone-for-review/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: what if I couldn't find anyone around for review/recommendation? Am I going to be eligible?
+type: "faq"
+priority: "41"
+tags: ["reviews", "recommendations", "3R"]
+---
+
+You will have enough people. Don't worry. You have to try and keep your eyes open. Learn to ask. Fail. And repeat.\
```

Comment:
give double line breaks to change lines. No need to give this backslash. see my entries. Remove this from all markdowns. No backslash.

Reply:
Done. Will keep in mind from next time.